### PR TITLE
Prepare for release v2.3.0

### PR DESCRIPTION
**Pull request by KB-perByte**

### CODE REVIEW

File: changelogs/.plugin-cache.yaml [hunk 1]
```diff
@@ -0,0 +1,217 @@
+objects:
```
Comment (NilashishC) [inline changelogs/.plugin-cache.yaml:1]: Can we add this in .gitignore?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,65 +4,91 @@ Cisco Ios Collection Release Notes
...
+
 v2.2.0
 ======
 
 Minor Changes
 -------------
-- Add ios_logging_global module. (https://github.com/ansible-collections/cisco.ios/pull/306).
-- Add Prefix list resource module (https://github.com/ansible-collections/cisco.ios/pull/308).
+
+- Add ios_logging_global module.
```
Comment (NilashishC) [inline CHANGELOG.rst:37]: There seems to be a _LOT_ of changes that are not related to this release. Was the antsibull-changelog tool not used for previous releases?

Comment (KB-perByte) [reply CHANGELOG.rst:37]: @NilashishC I am not sure, maybe it was not used. As the `collection_prep_add_docs` and `antsibull-changelog` produced a  a huge chunk of files

File: changelogs/.plugin-cache.yaml (hunk 1 above)
Comment (KB-perByte) [reply changelogs/.plugin-cache.yaml]: @NilashishC Thanks

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,65 +4,91 @@ Cisco Ios Collection Release Notes
...
 - To fix Cisco IOS no log issue and add ignore txt for 2.12 (https://github.com/ansible-collections/cisco.ios/pull/304).
 
 Bugfixes
 --------
-- To fix the wrong arg being passed in acls template function (https://github.com/ansible-collections/cisco.ios/pull/305).
 
-New Modules
------------
-
-- ios_route_maps - Manage the attributes of Route maps on Cisco IOS.
```
Comment (ashwini-mhatre) [inline CHANGELOG.rst:49]: why this is removed?

File: CHANGELOG.rst [hunk 4]
```diff
@@ -165,25 +195,9 @@ New Modules
 v1.0.3
 ======
 
-Release Summary
```
Comment (ashwini-mhatre) [inline CHANGELOG.rst:168]: the tool is removing this for iosxr also I manually added that again

File: CHANGELOG.rst [hunk 5]
```diff
@@ -165,25 +195,9 @@ New Modules
...
-Bugfixes
---------
-
-- To fix the issue where ios acls was complaining in absence of protocol option value (https://github.com/ansible-collections/cisco.ios/pull/124).
-- To fix IOS l2 interfaces for traceback error and merge operation not working as expected (https://github.com/ansible-collections/cisco.ios/pull/103).
-
 v1.0.2
 ======
 
-Release Summary
```
Comment (ashwini-mhatre) [inline CHANGELOG.rst:182]: added this manually for iosxr. I think this is bug from tool

File: changelogs/changelog.yaml [hunk 6]
```diff
@@ -208,39 +204,42 @@ releases:
...
   2.1.0:
     changes:
       bugfixes:
       - To fix the wrong arg being passed in acls template function (https://github.com/ansible-collections/cisco.ios/pull/305).
-      security_fixes:
-      - To fix Cisco IOS no log issue and add ignore txt for 2.12 (https://github.com/ansible-collections/cisco.ios/pull/304).
       minor_changes:
-      - Add support for ansible_network_resources key allows to fetch the available resources for a platform (https://github.com/ansible-collections/cisco.ios/pull/292).
       - Add ios_route_maps Resource Module (https://github.com/ansible-collections/cisco.ios/pull/297).
-      modules:
```
Comment (ashwini-mhatre) [inline changelogs/changelog.yaml:230]: why this is removed?

File: CHANGELOG.rst (hunk 3 above)
Comment (KB-perByte) [reply CHANGELOG.rst]: manually added back, thanks @ashwini-mhatre

File: CHANGELOG.rst (hunk 4 above)
Comment (KB-perByte) [reply CHANGELOG.rst]: manually added back @ashwini-mhatre thanks

**Review by justjais — APPROVED** [review]
